.github/workflows/build-pelican.yml

```diff
@@ -0,0 +1,13 @@
+name: Build site
```

Comment:
Could you add some comments to this file explaining what it is for (it does it make a preview?)

Reply:
Added brief note at the top. I don't think this is what generates the preview.